### ISSUE TITLE
[Feature] : Titanic Dataset Visualization

### ISSUE THREAD

**Issue by AKACHI-4** [OpenCodeEra/ML-X#5]
## What ?

I would like to add a data visualization notebook for the Titanic dataset to explore passenger demographics, survival rates, and class distribution aboard the Titanic.

- ### Dataset

describe the dataset you will use for this case.

## Code of Conduct

- [x] I agree to follow this project's Code of Conduct

<br>
<br>

> **Kindly assign it to me under _HacktoberFest'23_** 
> **I will surely complete it ASAP**

**Comment by Chandak-Keshav** [comment]
Hey, kindly assign me this issue @AKACHI-4 . I have already worked with titanic dataset previously and would be glad to help with it's visualization.

**Comment by AKACHI-4** [comment]
> Hey, kindly assign me this issue @AKACHI-4 . I have already worked with titanic dataset previously and would be glad to help with it's visualization.

Hey @Chandak-Keshav,

It's great to hear that you've already worked on it.

please provide the details of the dataset and go ahead.

**Comment by Chandak-Keshav** [comment]
https://www.kaggle.com/competitions/titanic/data
I will be using the above dataset for data visualization. Should I go ahead with this dataset or do you have any particular dataset in mind @AKACHI-4

**Comment by AKACHI-4** [comment]
> https://www.kaggle.com/competitions/titanic/data I will be using the above dataset for data visualization. Should I go ahead with this dataset or do you have any particular dataset in mind @AKACHI-4

Hey, @Chandak-Keshav.... the dataset is good, but I'll suggest finding another one with more parameters so that you can extract more meaning from the data.

**Comment by Chandak-Keshav** [comment]
https://www.kaggle.com/competitions/spaceship-titanic/data?select=train.csv
How about this dataset @AKACHI-4

**Comment by AKACHI-4** [comment]
@Chandak-Keshav, yes, go ahead... make sure to use libraries for better visualization to extract meaningful insights from it.
